datahub-web-react/src/app/entityV2/shared/components/styled/search/EmbeddedListSearchSection.tsx

```diff
@@ -146,6 +170,7 @@ export const EmbeddedListSearchSection = ({
 
     return (
         <EmbeddedListSearch
+            entityTypes={SEARCH_ENTITY_TYPES}
```

Comment:
Again, we've been managing this on the backend for some time. Why move to the UI layer?